package.json

```diff
@@ -12,41 +12,48 @@
     "@types/react-native-dotenv": "^0.2.0",
     "@types/react-native-i18n": "^2.0.0",
     "@types/react-redux": "^7.1.2",
-    "@types/styled-components": "^4.1.18",
+    "@types/styled-components": "^4.1.19",
```

Comment:
Auto increment dependencies
15/18/20/23/26/27/29/33/35/40/41/42/46/49/56 Lines.
Auto increment dependencies can crush you app in the future